metadata-ingestion/src/datahub/ingestion/source/data_lake_common/path_spec.py

```diff
@@ -150,6 +150,12 @@ class Config:
         description="The tables_filter_pattern configuration field uses regular expressions to filter the tables part of the Pathspec for ingestion, allowing fine-grained control over which tables are included or excluded based on specified patterns. The default setting allows all tables.",
     )
 
+    print_debug: bool = Field(
```

Comment:
Why not exporting `DATAHUB_DEBUG=1`? Unless we have very good reason for having this additional setting I think it is redundant.

Reply:
+1

what's the reason for introducing this new pattern which requires to enable two flags, the general debug flag and the `print_debug` one ?

```
        if self.print_debug:
            logger.debug(...)
```

Reply:
I will remove them. I don't think such an env var will ever be set :)